mathics/builtin/plot.py

```diff
@@ -1056,8 +1056,8 @@ def labels(names):
 class BarChart(_Chart):
     """
     <dl>
-    <dt>'BarChart[{$p1$, $p2$ ...}]'
-        <dd>draws a bar chart.
+        <dt>'BarChart[{$p1$, $p2$ ...}]'
+        <dd>makes a bar chart with lengths $b1$, $b2$, ....
```

Comment:
Shouldn't this be `$p1$, $p2$, ...`

Reply:
Oops - yes you are correct. Fixed in dcdf81a7